### DISCUSSION TITLE
extra_css doesn't seem to work

### DISCUSSION THREAD

**Discussion by pajaro1966** [squidfunk/mkdocs-material#7680] [Q&A] [answered]
I'm trying to change the h5 styles of my site. 

(i) I've created a extra.css file:
![image](https://github.com/user-attachments/assets/86a9343b-81d2-4506-90d7-e3dbf61259ff)

(ii) Then I've added the needed code to my MkDocs.yml

![image](https://github.com/user-attachments/assets/fb426adb-4b8d-43f9-bb87-ed4548375da7)

(iii) I've deleted the cache memory of my browser.

However, the style of my h5 remain the same:

![image](https://github.com/user-attachments/assets/3d5a18d4-fd9f-4533-b561-85e889c93f8b)

I've read all the threads about this issue that I could, but I must be missing something important. 
I would appreciate any help.

Best regards,

Antonio

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @pajaro1966,
the issue is [Specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), the Material theme defines a stronger CSS selector and your extra css is too weak to override it. Best to check in the browser tools what the issue is.

```css
.md-typeset h5 {
  text-transform: lowercase;
  color: red;
}
```

https://github.com/user-attachments/assets/1552e847-b9f0-44d3-9457-3d0d04cec731

**Reply by pajaro1966** [reply]
Thank you so much for your fast and clear answer. That was it. Sorry for such a basic question, but my area of expertise is Greek (ancient and modern): my technical knowledge is limited and I always forget such basic issues. I’m using material mkdocs to provide my students with my lessons’ notes and other grammatical knowledge. Sorry for wasting your time. I do really appreciate your help.

Antonio

**Reply by alexvoss** [reply]
It is a while ago now but I used to teach software engineering and was creating my lecture material using Material for MkDocs. Hats off for doing this with Greek!

**Reply by kamilkrzyskow** [reply]
Don't feel bad for asking basic questions, you asked a very specific question and provided proper screenshots and other explanation on what you expect to happen, the video took 1 minute to make (+ 5 minutes more to cut the size below 10MB for GitHub 😅), so it really wasn't a waste of time :v:

In general it's recommended to [create a minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) to help with removing user error, but the size of the problem didn't really require that, as it was easy to check with what was provided on the screenshots 😃
